addons/sale_pdf_quote_builder/views/sale_order_template_views.xml

```diff
@@ -25,7 +26,28 @@
                         The pdf of your quotes will be built by putting together header pages, product descriptions,
                         details of the quote and then the footer pages. <br/>
                         If empty, it will use those define in the company settings. <br/>
-                        <widget name="documentation_link" path="/_downloads/5f0840ed187116c425fdac2ab4b592e1/pdfquotebuilderexamples.zip" label=" Download examples" icon="fa-arrow-right"/>
+                        <div class="d-flex flex-column">
+                            <widget
+                                name="documentation_link"
+                                path="/_downloads/5f0840ed187116c425fdac2ab4b592e1/pdfquotebuilderexamples.zip"
+                                label=" Download examples"
+                                icon="fa-arrow-right"
+                            />
+                            <a
+                                href="https://www.youtube.com/watch?v=M_95g8_dhGk"
+                                target="_blank"
+                                invisible="not quotation_document_ids"
+                            >
+                                <i class="fa fa-arrow-right"/>Link to the explainer video
+                            </a>
```

Comment:
![image](https://github.com/user-attachments/assets/ac089463-7ba3-4e4e-adfb-0b3bf103a7bb)
Could you add a space before the 'Link to the explainer video' so that the display is consistent between the two please ?
(and replace `explainer` by `demo`, doesn't seem to be an usual english word).